### ISSUE TITLE
auto tag completion

### ISSUE THREAD

**Issue by GoogleCodeExporter** [aymanhs/jsyntaxpane#9]
```
your jsyntaxpane is very cool, but recently I focus on my own XML editor,
which can use your library. but a shortcoming is tag auto completion, e.g.

when I press <Test>, another </Test> should be automatically followed
because of it is an XML document.

Could you please fix it?


```

Original issue reported on code.google.com by `hongyi.j...@gmail.com` on 30 Jul 2008 at 10:10

**Comment by GoogleCodeExporter** [comment]
```
This is an Enhancement.  I will investigate implementing this as an Action for 
XML
editing for a future release.
```

Original comment by `ayman.al...@gmail.com` on 31 Jul 2008 at 1:34
- Changed title: **auto tag completion**
- Changed state: **Started**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ayman.al...@gmail.com` on 31 Jul 2008 at 1:41

**Comment by GoogleCodeExporter** [comment]
```
Preliminary completions actions are added for Java.  The completions are in a
properties file in the META-INF\services folder.
```

Original comment by `ayman.al...@gmail.com` on 21 Sep 2008 at 12:33

**Comment by GoogleCodeExporter** [comment]
```
Closing issue as fixed in 0.9.3.
```

Original comment by `ayman.al...@gmail.com` on 16 Nov 2008 at 10:40
- Changed state: **Fixed**
